src/pages/_app.tsx

```diff
@@ -61,10 +58,10 @@ const config = createConfig({
                 projectId: projectId,
                 qrModalOptions: {
                     themeVariables: {
-                        '--w3m-font-family':
+                        '--wcm-font-family':
```

Comment:
w3m is deprecated and no longer maintained, wagmi supports wcm. Let me discuss this with @biga816

Reply:
Because `wagmi` supports `w3m`, not wcm`, on the version we use now.
You can check it from the interface file of `createConfig`.

Reply:
![Cursor_and_types_d_ts_—_secured-finance-app](https://github.com/Secured-Finance/secured-finance-app/assets/10794545/245fe774-eef8-4872-8550-b6e7d829d332)

Reply:
As discussed with Aki,

w3m is supported in the earlier versions of wagmi, this is the wagmi dep on develop
secured-finance-web@1.3.1 /Users/haibara/secured-finance-app
├─┬ @0xsquid/widget@1.6.20
│ └── wagmi@0.12.19
└── wagmi@1.4.12

and this is the wagmi dep on SF-727

secured-finance-web@1.3.1 /Users/haibara/secured-finance-app
├─┬ @0xsquid/widget@1.6.20
│ └── wagmi@0.12.19
└── wagmi@1.4.13

v1.4.13 and higher versions support wcm, w3m is deprecated. For now, I will install v1.4.12 since I'm not upgrading wagmi in SF-727